java/ingest/ingest-batcher-core/src/main/java/sleeper/ingest/batcher/IngestBatcher.java

```diff
@@ -76,27 +76,59 @@ public static Builder builder() {
-        int minFiles = properties.getInt(INGEST_BATCHER_MIN_JOB_FILES);
-        long minBytes = properties.getBytes(INGEST_BATCHER_MIN_JOB_SIZE);
-        Instant maxReceivedTime = time.minus(Duration.ofSeconds(
-                properties.getInt(INGEST_BATCHER_MAX_FILE_AGE_SECONDS)));
-        if ((inputFiles.size() >= minFiles &&
-                totalBytes(inputFiles) >= minBytes)
-                || inputFiles.stream().anyMatch(file -> file.getReceivedTime().isBefore(maxReceivedTime))) {
+        long totalBytes = totalBytes(inputFiles);
+        if (shouldCreateBatches(properties, inputFiles, time)) {
             BatchIngestMode batchIngestMode = batchIngestMode(properties).orElse(null);
-            LOGGER.info("Creating batches for {} input files", inputFiles.size());
+            LOGGER.info("Creating batches for {} files with total bytes of {}", inputFiles.size(), totalBytes);
+            List<Instant> receivedTimes = inputFiles.stream()
+                    .map(FileIngestRequest::getReceivedTime)
+                    .sorted().collect(toList());
+            LOGGER.info("Files to batch were received between {} and {}",
+                    receivedTimes.get(0), receivedTimes.get(receivedTimes.size() - 1));
             createBatches(properties, inputFiles)
                     .forEach(batch -> sendBatch(tableName, batchIngestMode, batch));
         }
     }
 
+    private boolean shouldCreateBatches(
+            TableProperties properties, List<FileIngestRequest> inputFiles, Instant time) {
+        int minFiles = properties.getInt(INGEST_BATCHER_MIN_JOB_FILES);
+        long minBytes = properties.getBytes(INGEST_BATCHER_MIN_JOB_SIZE);
+        int maxAgeInSeconds = properties.getInt(INGEST_BATCHER_MAX_FILE_AGE_SECONDS);
+        Instant maxReceivedTime = time.minus(Duration.ofSeconds(maxAgeInSeconds));
+        long totalBytes = totalBytes(inputFiles);
+        boolean maxAgeMet = inputFiles.stream().anyMatch(file -> file.getReceivedTime().isBefore(maxReceivedTime));
+        if (maxAgeMet) {
+            LOGGER.info("At least one file has reached the maximum age of {} seconds", maxAgeInSeconds);
+            return true;
+        }
+        boolean meetsMinFiles = true;
+        if (inputFiles.size() < minFiles) {
+            LOGGER.info("Number of files ({}) does not satisfy minimum file count for job ({})",
+                    inputFiles.size(), minFiles);
+            meetsMinFiles = false;
+        }
+        if (totalBytes < minBytes) {
+            LOGGER.info("Total bytes for files ({}) does not satisfy minimum size for job ({})",
+                    totalBytes, minBytes);
+            meetsMinFiles = false;
+        }
+        return meetsMinFiles;
+    }
+
     private void sendBatch(String tableName, BatchIngestMode batchIngestMode, List<FileIngestRequest> batch) {
         String jobId = jobIdSupplier.get();
         List<String> files = store.assignJobGetAssigned(jobId, batch);
```

Comment:
In the log line in this method where we log `Sending ingest job with {} files to {}` can we also log the total number of bytes in the files?